addons/medical_treatment/functions/fnc_placeInGrave.sqf

```diff
@@ -23,9 +23,9 @@ if ((alive _patient) && {GVAR(allowGraveDigging) < 2}) exitWith {
     [LSTRING(bodybagWhileStillAlive)] call EFUNC(common,displayTextStructured);
 };
 
-private _graveClassname = missionNameSpace getVariable [QGVAR(graveClassname), "Land_Grave_dirt_F"];
-// Land_Grave_dirt_F needs to be rotated 90 degrees to line up with the body
-private _graveRotation = missionNameSpace getVariable [QGVAR(graveRotation), 90]; 
+private _graveClassname = missionNameSpace getVariable [QGVAR(graveClassname), "Land_ACE_Grave"];
+// Land_Grave_dirt_F needed to be rotated 90 degrees to line up with the body, Land_ACE_Grave does not.
```

Comment:
```suggestion
// Land_Grave_dirt_F needs to be rotated 90 degrees to line up with the body, Land_ACE_Grave does not.
```

Reply:
This new model might not need the rotation, in which case the entire comment can be removed instead.

Reply:
I'm not a programmer. I took a cursory look and it looked like `_graveRotation` was still needed as a parameter. I can remove the comment though, yeah.